cmd/relay/config/load.go

```diff
@@ -156,8 +156,8 @@ func LoadClaimService(mt *merkletree.MerkleTree, rootservice rootsrv.Service, ks
 	return claimsrv.New(mt, rootservice, signsrv.New(ks, acc))
 }
 
-func LoadNameService(identityservice identitysrv.Service, claimservice claimsrv.Service, ks *keystore.KeyStore, acc accounts.Account, domain string, namespace string) namesrv.Service {
-	return namesrv.New(claimservice, identityservice, signsrv.New(ks, acc), domain)
+func LoadNameService(claimservice claimsrv.Service, ks *keystore.KeyStore, acc accounts.Account, domain string, namespace string) namesrv.Service {
```

Comment:
Why is the relay loading the name service?

Reply:
right, removed